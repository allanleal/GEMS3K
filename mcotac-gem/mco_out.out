 node2 and rest
0.649678752723E-05 0.124503208894E-03 0.288396656996E-08 0.971294275435E-07 0.224086229367E-07 0.373466857428E-04 0.871312362241E-04 0.000000000000E+00 0.200000002337E-06 0.000000000000E+00 0.100003800612E-04 0.871761884599E-04 0.119557957009E-09 0.553417610552E+02 0.000000000000E+00 0.199999991607E-03 0.000000000000E+00
0.330999988364E-03 0.330999988364E-03 0.200000002337E-06 0.110683696418E+03 0.100000001169E-06 0.553428612088E+02 0.000000000000E+00
 end node 2 and rest
  bn n=2  
0.12E-03 0.37E-04 0.97E-07 0.20E-06 0.87E-04 0.55E+02 0.10E-14
  cn n=2  
0.12E-09 0.65E-05 0.29E-08 0.87E-04 0.22E-07 0.00E+00 0.00E+00 0.10E-04
  pn n=2  
0.00E+00 0.20E-03 0.00E+00
  2 p_xDc(ib) 
0.65E-05 0.12E-03 0.29E-08 0.97E-07 0.22E-07 0.37E-04 0.87E-04 0.00E+00 0.20E-06 0.00E+00 0.10E-04 0.87E-04 0.12E-09
0.55E+02 0.00E+00 0.20E-03 0.00E+00
  end of initial calc 
