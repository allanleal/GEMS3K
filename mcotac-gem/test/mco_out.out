 node2 and rest
